docs/tutorials/course/fairbo.md

```diff
@@ -0,0 +1,506 @@
+# Fair Bayesian Optimization
+
+Given the increasing importance of machine learning (ML) in our lives, with a wide use of automated systems in several domains, such as financial lending, hiring, criminal justice, and college admissions, there has been a major concern for ML to unintentionally encode societal biases and result in systematic discrimination. For example, a classifier that is only tuned to maximize prediction accuracy may unfairly predict a high credit risk for some subgroups of the population applying for a loan. 
+
+In several real-world domains, accuracy is not the only objective of interest. The model should simultaneously give guarantees on other important aspects. Algorithmic fairness tries to find algorithms that not only keep a high level of accuracy but also enforce a certain level of fairness and avoid biases.
+
+The approach in this tutorial is based on the paper "Fair Bayesian Optimization". Link: https://arxiv.org/abs/2006.05109
+
+If you use this code please cite 'Fair Bayesian Optimization'. Valerio Perrone, Michele Donini, Muhammad Bilal Zafar, Robin Schmucker, Krishnaram Kenthapadi, Cédric Archambeau. AIES. 2021.
+
+In this tutorial we are going to use the German Credit Data (1). This dataset contains a binary classification task, where the goal is to predict if a person has "good" or "bad" credit risk. The dataset contains 20 attributes, including a sensitive features such as the (binary) gender of the person. In the following we download, load and explore this dataset.
+
+(1) Dua, D. and Graff, C. (2019). [UCI Machine Learning Repository](http://archive.ics.uci.edu/ml). Irvine, CA: University of California, School of Information and Computer Science.
```

Comment:
why is this here? I'd just provide the link to the dataset itself as you've already done below.